reana.yaml

```diff
@@ -1,26 +1,28 @@
 version: 0.1.0
 metadata:
-  - authors:
-    - Harri Hirvonsalo <hjhsalo@gmail.com>
-    - Diego Rodriguez <diego.rodriguez@cern.ch>
-    - Tibor Simko <tibor.simko@cern.ch>
-  - title: Hello world - A simple reusable analysis example
-  - date: 18 January 2017
-  - repository: https://github.com/reanahub/reana-demo-helloworld/
+  authors:
+   - Harri Hirvonsalo <hjhsalo@gmail.com>
+   - Diego Rodriguez <diego.rodriguez@cern.ch>
+   - Tibor Simko <tibor.simko@cern.ch>
+  title: Hello world - A simple reusable analysis example
+  date: 18 January 2017
+  repository: https://github.com/reanahub/reana-demo-helloworld/
 code:
-  - files:
-    - code/helloworld.py
+  files:
+   - code/helloworld.py
 inputs:
-  - files:
+  files:
     - inputs/names.txt
-  - parameters:
-    - sleeptime: 2
+  parameters:
```

Comment:
Hmm, should we make `parameters` to be a list? What if somebody using some other workflow engine needs multiple options? (`foo -n 10 -n 13`) Talk IRL.